obspy/core/tests/test_waveform_plugins.py

```diff
@@ -81,7 +81,10 @@ def test_read_and_write(self):
                     tr.stats.location = "00"
                     tr.stats.channel = "EHE"
                     tr.stats.calib = 0.199999
-                    tr.stats.delta = 0.005
+                    if format == 'AH':
```

Comment:
Yes I got that from the conversion. But maybe just change the delta to 0.25 (or 0.001 which can also be exactly represented by 32 bit floats) for all formats? That would reduce the complexity of this test.